common/static/common/js/spec/components/paging_spec.js

```diff
@@ -73,19 +73,19 @@ define([ "jquery", "common/js/spec_helpers/ajax_helpers", "URI",
                 describe("setPage", function () {
                     it('can set the current page', function () {
                         var requests = AjaxHelpers.requests(this);
-                        pagingView.setPage(0);
+                        pagingView.setPage(1);
```

Comment:
I'm slightly confused on these tests -- we set the page to 1, respond with some fake data, and then expect the current page to be 0? They're passing, so I guess I'm missing something, I just don't see why.